src/components/Sing/SingEditor.vue

```diff
@@ -57,16 +61,49 @@ const cancelExport = () => {
   store.dispatch("CANCEL_AUDIO_EXPORT");
 };
 
-// エンジン初期化後の処理
-const unwatchIsEnginesReady = watch(
-  // TODO: 最初に１度だけ実行している。Vueっぽくないので解体する
-  () => props.isEnginesReady,
-  async (isEnginesReady) => {
-    if (!isEnginesReady) return;
+const isCompletedInitialStartup = ref(false);
+// TODO: Vueっぽくないので解体する
+onetimeWatch(
+  () => props.isProjectFileLoaded,
+  async (isProjectFileLoaded) => {
+    if (isProjectFileLoaded == "waiting" || !props.isEnginesReady) return false;
```

Comment:
return falseが何を意味するかちょっと分かりづらいかも･･･？
`watch続行 | watchやめる`みたいなLiteral string 返す設計が分かりやすそうなのですが、なんか良い名称思いつかなかったので、コメント書くとかどうでしょう。

```suggestion
    if (isProjectFileLoaded == "waiting" || !props.isEnginesReady) return false; // watch続行
    // TODO: return true側にもコメント書く
    // TODO: トーク側にもコメント書く
```

ちょっと分かりにくいかもだけど`continue` | `unwatch`とかでも良いかも